checklists/alerts.md

```diff
@@ -0,0 +1,54 @@
+- [ ] Database transaction time - [How to create this alert](https://docs.newrelic.com/docs/alerts/new-relic-alerts/defining-conditions/create-alert-conditions)
+- [ ] Error percentage - [How to create this alert](https://docs.newrelic.com/docs/alerts/new-relic-alerts/defining-conditions/create-alert-conditions)
+- [ ] Key transactions response time - [How to create this alert](https://docs.newrelic.com/docs/apm/transactions/key-transactions/view-key-transactions-alert-information)
+- [ ] Key transactions error percentage - [How to create this alert](https://docs.newrelic.com/docs/apm/transactions/key-transactions/view-key-transactions-alert-information)
+- [ ] Third-party services (redundant with Uptime Robot)
+- [ ] Application web transaction time deviation baseline - [How to create this alert](https://docs.newrelic.com/docs/alerts/new-relic-alerts/defining-conditions/create-baseline-alert-conditions)
+- [ ] Application transaction database time deviation baseline - [How to create this alert](https://docs.newrelic.com/docs/alerts/new-relic-alerts/defining-conditions/create-baseline-alert-conditions)
+
+## Uptime Robot
+
+- [ ] Django web application (production and staging) - [How to create this alert](https://blog.uptimerobot.com/need-only-the-down-or-up-notifications-you-are-covered/)
+- [ ] Celery Flower (production and staging) - [How to create this alert](https://blog.uptimerobot.com/need-only-the-down-or-up-notifications-you-are-covered/)
+- [ ] Frontend application (production and staging) - [How to create this alert](https://blog.uptimerobot.com/need-only-the-down-or-up-notifications-you-are-covered/)
+- [ ] Third-party services - [How to create this alert](https://blog.uptimerobot.com/need-only-the-down-or-up-notifications-you-are-covered/)
+
+## CircleCI
+
+- [ ] Build failure - [How to create this alert](https://circleci.com/docs/2.0/notifications/)
+
+## Codecov
+
+- [ ] Coverage change on pull requests - [How to create this alert](https://docs.codecov.io/docs/notifications)
+- [ ] Coverage change by direct commits to master branch that might not be coming from pull-requests - [How to create this alert](https://docs.codecov.io/docs/notifications)
+
+## CodeClimate
+
+- [ ] Code issues - [How to create this alert](https://docs.codeclimate.com/docs/user-email-preferences)
+
+## Sqreen
+
+- [ ] Incidents - [How to create this alert](https://docs.sqreen.com/using-sqreen/incidents/#notifications)
+
+## Monit
+
+- [ ] Machine status change (start, stop, restart) - [How to create this alert](https://mmonit.com/monit/documentation/monit.html#System)
+- [ ] Memory usage - [How to create this alert](https://mmonit.com/monit/documentation/monit.html#System)
+- [ ] CPU usage - [How to create this alert](https://mmonit.com/monit/documentation/monit.html#System)
+- [ ] Disk usage - [How to create this alert](https://mmonit.com/monit/documentation/monit.html#System)
```

Comment:
Seems there are a number repeated external links. Could you please grab the repeated ones and use a reference like `[1]`? Then add a section below "How to set alerts" and list the references 1, 2, 3, etc. In case you don't understand, please talk to me in person. Thanks!

Reply:
Only for the repeated ones or all of them?

Reply:
all of them, I think it's better to read. Thanks!